etc/ReplayOfflineConfiguration.py

```diff
@@ -44,6 +45,11 @@
 # Use this in order to limit the number of lumisections to process
 #setInjectLimit(tier0Config, 10)
 
+# Define streams to ignore. These wont be injected
+setHelperAgentStreams(tier0Config, {'SecondAgent': ["ScoutingPF", "L1ScoutingSelection", "L1Scouting", "Express"],
```

Comment:
Here we provide a dictionary whose keys are agent names (arbitrary).
If a key is 'SecondAgent' then, to process the respective streams, we need
```
config.Tier0Feeder.agentName = "SecondAgent"
```

Also, name is arbitrary. We only need the name to match whatever we write in the config.py file. If there is no match, nothing is processed.